gateway-server/src/main/java/org/apache/knox/gateway/services/topology/impl/DefaultTopologyService.java

```diff
@@ -193,7 +193,6 @@ private void redeployTopology(Topology topology) {
             } else {
               Thread.sleep(10);
               elapsed = System.currentTimeMillis() - start;
-              continue;
```

Comment:
Just curious, why was continue removed?

Reply:
Unnecessary as last statement in loop

Reply:
Just a clean-up thing while I was in there

Reply:
ah gotcha.